bottle_server.py

```diff
@@ -36,14 +36,14 @@
 app = Bottle()
 default_app.push(app)
 
-import StringIO # NB: don't use cStringIO since it doesn't support unicode!!!
+import io # NB: don't use cStringIO since it doesn't support unicode!!!
 import json
```

Comment:
In the README, explain how to update bottle.py and that no local modifications of that file should be done.

Reply:
I'm not sure I completely understand, but I updated the README.md, please let me know if there is something more you want me to add.